dags/operational_monitoring.py

```diff
@@ -0,0 +1,51 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from utils.gcp import gke_command
+
+from airflow.operators.sensors import ExternalTaskSensor
+
+
+default_args = {
+    "owner": "msamuel@mozilla.com",
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "msamuel@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2021, 6, 3),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+PROJECT_ID = "operational_monitoring"
```

Comment:
This probably should be something like `DAG_NAME`. `PROJECT_ID` could be confusing here since it doesn't actually refer to a GCP project.

Reply:
good point